tendrl/ceph_integration/manager/cluster_monitor.py

```diff
@@ -366,6 +367,122 @@ def on_sync_object(self, data):
                 " received for %s" % data['type']
             )
 
+    def _get_utilization_data(self):
+        from ceph_argparse import json_command
```

Comment:
Please use https://github.com/Tendrl/ceph-integration/blob/master/tendrl/ceph_integration/ceph.py

It already contains utility methods for calling RADOS etc

rest LGTM